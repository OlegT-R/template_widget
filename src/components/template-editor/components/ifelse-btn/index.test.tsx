import React from 'react';

import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { IfElseBtn } from '.';

const onClick = () => null;
test('<IfElseBtn />', () => {
  const component: any = shallow(<IfElseBtn onClick={onClick} />);
  expect(toJson(component)).toMatchSnapshot();
});
