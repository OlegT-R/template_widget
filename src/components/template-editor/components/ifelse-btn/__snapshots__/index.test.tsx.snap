// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IfElseBtn /> 1`] = `
<Button
  block={false}
  ghost={false}
  htmlType="button"
  loading={false}
  onClick={[Function]}
  type="dashed"
>
  Click to Add: 
  <styled.span>
    IF
  </styled.span>
  {{some_variable} or expression}
  <styled.span>
    THEN
  </styled.span>
  {then_value}
  <styled.span>
    ELSE
  </styled.span>
  {else_value}
</Button>
`;
